### PR TITLE
♻️  Renamed Switch Button to Toggle [CB-15862]

### PR DESCRIPTION
**Pull request by thaoms**
There is a second PR needed to replace the occurrences in Engage of course.

Fixes CB-15862

### CODE REVIEW

File: src/Buttons/Toggle.tsx [hunk 1]
```diff
@@ -1,28 +1,28 @@
 import '@clarabridge/unified-ui/dist/switch-buttons.css';
```
Comment (cedricengagor) [inline src/Buttons/Toggle.tsx:1]: Q: Can you also open a PR on `unified-ui` to rename this to switch-btn together with it's classnames?

Comment (thaoms) [reply src/Buttons/Toggle.tsx:1]: This is the last time I do a rename after a feature is made..

**Review by matthias-deschoenmacker — APPROVED** [review]

**Comment by thaoms** [comment]
This one need sot be merged first + package update should then be included in this PR
https://github.com/ClarabridgeInc/unified-ui/pull/100

**Review by cedricengagor — CHANGES_REQUESTED** [review]
REQ: You still need to install the package version increase. Test is indicating that 🙇

**Review by cedricengagor — APPROVED** [review]

**Pull request merged**